docs/src/_guide/advanced-topics.md

```diff
@@ -13,3 +13,159 @@ Most of Meltano's features are available without installing any additional packa
+
+### What is the recommended way to install and use these connectors?
+
+Any connector that is listed on MeltanoHub as being maintained by `airbyte` can be installed and run as you would any other connector. Use `meltano add extractor &lt;tap>` to add and install.
+
+### What are the limitations of running Airbyte-based connectors with Meltano?
+
+Based on the current implementation and testing, the main challenge is putting these connectors into production with Meltano. See the next question for more details on this.
+
+### Am I able to put my Meltano project with Airbyte connectors in production?
+
+Yes, assuming you can set up Docker-in-docker. Since each Airbyte connector is itself a Docker image, running inside a dockerized Meltano project would require “docker in docker” which has some possible challenges around privileges and permissions on certain systems.
+
+Within Codespaces, adding support for docker-in-docker was as easy as [adding a few lines](https://github.com/meltano/meltano-codespace-ready/commit/5ffb9fb6e3232142c8e2307340c0a4fb66379db4) to the devcontainer.json file.
+
+It is also possible to run [Meltano on GitHub Actions](https://github.com/brooklyn-data/meltano-on-github-actions). It’s likely possible to update this action to supporting docker-in-docker as well.
+
+There [are](zhsj.me/blog/view/dind-without-privileged) [several](shisho.dev/blog/posts/docker-in-docker) [articles](docs.docker.com/engine/security/rootless) on the web which discuss docker-in-docker in more detail.
+
+We are interested in exploring this challenge with the community. Please share in **this discussion or connect with us in Slack**about your experiences with attempting to put this into production.
+
+### How does this work with custom connectors made with the Airbyte CDK?
+
+This integration will work with any dockerized Airbyte source, whether it was made with their CDK or not.
+
+To configure your Airbyte connector as a [custom plugin]([https://docs.meltano.com/concepts/project#custom-plugin-definitions](https://docs.meltano.com/concepts/project#custom-plugin-definitions)) in Meltano you can copy the configuration shown below into your meltano.yml file then replace `name`, and the `value` for the airbyte_spec.image to reference your docker image. You can configure your connector without defining all of the settings and their metadata.
+
+```yaml
+ - name: tap-pokeapi # REPLACE THIS WITH YOUR CONNECTOR NAME
+   variant: airbyte
+   executable: tap-airbyte
+   namespace: tap_airbyte
+   pip_url: git+https://github.com/z3z1ma/tap-airbyte.git
+   capabilities:
+   - catalog
+   - state
+   - discover
+   - about
+   - stream-maps
+   - schema-flattening
+   settings:
+   - description: Airbyte image to run
+     kind: string
+     label: Airbyte Spec Image
+     name: airbyte_spec.image
+     value: airbyte/source-punk-api # REPLACE THIS WITH YOUR IMAGE NAME
```

Comment:
```suggestion
     value: airbyte/source-pokeapi # REPLACE THIS WITH YOUR IMAGE NAME
```

This was my fault but its probably less confusing if the example tap name and docker image match.